R/reporter.R

```diff
@@ -14,8 +14,8 @@ default_reporter <- function() {
   if (cli::is_ansi_tty()) {
     reporter("ansi_tty")
   } else if (cli::is_dynamic_tty()) {
-    stop("dynamic tty reporter not yet available")
-    reporter("dynamic_tty")
+    message("dynamic tty reporter not yet available")
```

Comment:
```suggestion
```

I don't think it's worth having a message. To a typical user, it's unreasonable to assume they would know what a "dynamic_tty" is supposed to mean.